src/client/datascience/jupyter/kernels/kernel.ts

```diff
@@ -611,7 +609,7 @@ export class Kernel implements IKernel {
     }
 }
 
-export async function executeSilently(session: IJupyterSession, code: string): Promise<nbformat.IOutput[]> {
+export async function executeSilently(session: Pick<JupyterKernel.IKernelConnection, 'requestExecute'>, code: string): Promise<nbformat.IOutput[]> {
```

Comment:
We're using Jupyter lab API types directly, not IJupyterSession (makes it easier to re-factor later on to use pure Jupyter npm API)